core/templates/dev/head/pages/exploration_player/AudioTranslationLanguageServiceSpec.js

```diff
@@ -53,26 +53,24 @@ describe('Audio translation language service', function() {
   it('should initialize the current audio language when ' +
      'no preferred language is set and the exploration contains ' +
      'an audio language that is related to the exploration language',
-    function() {
-      var allAudioLanguageCodesInExploration = ['hi-en', 'en'];
-      var preferredLanguageCode = null;
-      var explorationLanguageCode = 'hi';
-      atls.init(allAudioLanguageCodesInExploration, preferredLanguageCode,
-        explorationLanguageCode);
-      expect(atls.getCurrentAudioLanguageCode()).toEqual('hi-en');
-    }
-  );
+  function() {
```

Comment:
Sometimes we indent this by 2, sometimes we don't (see above). Which do we want to do? Let's just pick a standard and stick to it.

Personally I'd go with the indent-by-2, but as long as we're consistent I don't mind too much.